views/metrics/index.tsx

```diff
@@ -23,13 +24,22 @@ const Metrics: FC = () => (
     </Typography>
     <Box
       variant="container"
-      gap={['xs', 'xs', 'xs']}
-      gridTemplateColumns={['1fr', '1fr', '1fr', '1fr 1fr']}
+      gap={['xs', 'xs', 'xs', 'xs']}
+      gridTemplateColumns={['1fr', '1fr', '1fr', '1fr']}
     >
+      <InfoCards />
       <TotalLiquidity />
       <DailyVolume />
-      <ActiveWallets />
-      <TVLPools />
+      <Box
+        width="100%"
+        display="flex"
+        flexDirection={['column', 'column', 'row', 'row']}
+        justifyContent="center"
+        gap={['xs', 'xs', 'xs']}
```

Comment:
only `gap="xs"` its enought

Reply:
If I remove this value the layout will break, cuz I'm forcing to keep the same size for all break point.

Reply:
not make sense this affirmation